modules/_labs/header/react/spec/SearchBar.spec.tsx

```diff
@@ -12,41 +12,39 @@ describe('Header Search', () => {
     cb.mockReset();
   });
 
-  test('Searching empty string should focus input', async () => {
+  test('Searching empty string should focus input', () => {
     const inputLabelText = `label`;
-    const {getByLabelText} = render(<SearchBar onSubmit={cb} inputLabel={inputLabelText} />);
+    render(<SearchBar onSubmit={cb} inputLabel={inputLabelText} />);
 
-    fireEvent.submit(await getByLabelText(inputLabelText));
+    fireEvent.submit(screen.getByRole('search', {name: inputLabelText}));
```

Comment:
`@testing-library/dom` was updated to complain that multiple elements where found with this label. `findByRole` is restricting which element we're talking about.